src/common/store/storage/actions-background.ts

```diff
@@ -8,20 +7,17 @@ import dashboardMessaging from '../../../background/dashboard-messaging';
 export function userLoggedIn(auth) {
   return (dispatch, state) => {
     dispatch(userDataNew(auth));
-    dashboardMessaging.sendLoginData();
   };
 }
 
 export function accessTokenRefresh(auth) {
   return (dispatch, state) => {
     dispatch(userTokensRefreshed(auth));
-    dashboardMessaging.sendLoginData();
   };
 }
 
 export function userLoggedOut() {
   return (dispatch, state) => {
     dispatch(userDataCleared());
-    dashboardMessaging.sendLoginData();
   };
```

Comment:
These thunks are no more needed and don't even need to be thunks. I'll remove them in one of the upcoming PRs